packages/ui/src/theme/overwrites/components/textarea.ts

```diff
@@ -0,0 +1,37 @@
+export const Textarea = {
+  baseStyle: {
```

Comment:
I think we shouldn't define the default width and height for the Textarea. We would probably never need it exactly in this size so size should be defined only for a particular use cases